packages/in_app_purchase/ios/Tests/InAppPurchasePluginTest.m

```diff
@@ -0,0 +1,206 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
These test files were moved from the example/ project to the new Tests/ directory.  Added symlinks from the old location so the tests still pass in the example app.

Reply:
Turns out Xcode really didn't link the symlinking.  I changed the Xcode project to just point at the new location instead of following the symlinks.